requirements.txt

```diff
@@ -5,3 +5,4 @@ Twisted>=11.1.0
 ipaddress>=1.0.16
 zope.interface>=3.6.1
 txsocksx>=1.13.0
+pyopenssl>=16.1.0
```

Comment:
Doesn't this get added by Twisted? Or is that an optional thing? (I mean: I don't think txtorcon actually directly depends on pyopenssl, does it?)